contracts/staking/contracts/src/immutable/MixinStorage.sol

```diff
@@ -47,6 +47,10 @@ contract MixinStorage is
     // address for read-only proxy to call
     address internal readOnlyProxyCallee;
 
+    // mapping from StakeStatus to the total amount of stake in that status for the entire
+    // staking system.
+    mapping (uint8 => uint256) public globalStakeByStatus;
```

Comment:
We'll need to add this to `TestStorageLayout`

Reply:
`TestStorageLayout` is officially ded as of [2168](https://github.com/0xProject/0x-monorepo/pull/2168)